view/templates/settings/settings.tpl

```diff
@@ -196,6 +196,17 @@
 </div>
 </div>
 
+<h3 class="settings-heading"><a href="javascript:;">{{$importcontact}}</a></h3>
+<div class="settings-content-block">
+<input type="hidden" name="MAX_FILE_SIZE" value="30720" />
```

Comment:
Maybe a nitpicking part, but wouldn't it be better to have a `const MAX_FILE_SIZE = 30720;` inside `src\Module\Settings\*` and reference it here?

* its better visible when adapting the code
* you've just to define it once and can use it at both places (avoid forgetting one place or another when adapting the value)

Reply:
Possible yes. The admin should most likely the one controling that value via a configuration setting.

Reply:
Do you want to add it to this PR or shall I go on merging? :)

Reply:
I've *just* pushed it.